core/db/migrate/20150314013438_add_missing_indexes_on_spree_tables.rb

```diff
@@ -0,0 +1,57 @@
+class AddMissingIndexesOnSpreeTables < ActiveRecord::Migration
+  def change
+    unless index_exists?(:spree_promotion_rules_users, [:user_id, :promotion_rule_id])
+      add_index   :spree_promotion_rules_users,
```

Comment:
@arpitchauhan Yes, but we can put `name` into `options` hash (4th example).
If you check source, you will see that name will be consider too.